### PR TITLE
docs: add instructions for extracting gradle projects

### PR DESCRIPTION
**Pull request by justbuchanan**

### CODE REVIEW

File: kythe/web/site/examples.md [hunk 1]
```diff
@@ -113,6 +113,90 @@ $ /opt/kythe/tools/runextractor cmake \
...
+  {
+    "pattern": "(.*)",
+    "vname": {
+      "corpus": "MY_CORPUS",
+      "path": "@1@"
+    }
+  }
+]
+```
+
```
Comment (schroederc) [inline kythe/web/site/examples.md]: Maybe comment about using the new `vnames` tool to manually test the file?

File: kythe/web/site/examples.md [hunk 2]
```diff
@@ -113,6 +113,90 @@ $ /opt/kythe/tools/runextractor cmake \
...
+2. VName configuration
+
+Next, you will need to create a vnames.json mapping file, which tells the
+extractor how to assign vnames to files based on their paths. A basic vnames
+config for a gradle project looks like:
+
+```json
+// vnames.json
```
Comment (shahms) [inline kythe/web/site/examples.md]: can we ship a reasonable default file or template for this?

Comment (justbuchanan) [reply kythe/web/site/examples.md]: I agree we should. I think a good default template would look just like this, but have the java extractor replace the corpus value with something that can be specified via environment variable or flag. We currently do that for bazel, but I don't think the java extractor supports it (yet).

Comment (shahms) [reply kythe/web/site/examples.md]: It looks like the Java extractor only uses `KYTHE_CORPUS` is `KYTHE_VNAMES` is absent, but should probably use `KYTHE_CORPUS` if the corpus resulting from applying the vnames is empty.

Comment (justbuchanan) [reply kythe/web/site/examples.md]: I filed this as a separate issue: https://github.com/kythe/kythe/issues/4389.

Comment (justbuchanan) [reply kythe/web/site/examples.md]: Also #4390.

File: kythe/web/site/examples.md (hunk 1 above)
Comment (justbuchanan) [reply kythe/web/site/examples.md]: Done.

**Comment by justbuchanan** [comment]
Website is updated: https://kythe.io/examples/#extracting-gradle-based-repositories.

**Comment by creachadair** [comment]
> Website is updated: https://kythe.io/examples/#extracting-gradle-based-repositories.

It's a fairly minor thing, but there seems to be some asciidoc wonkiness in that section: All the instruction points are numbered "1." and their content isn't co-indented with them. Maybe we should switch to subsection headers instead?

**Comment by justbuchanan** [comment]
Thanks for catching that. Not sure why it doesn't like my numbers, but subsection headers work for me. Here's a fix: https://github.com/kythe/kythe/pull/4391.

**Pull request merged**